src/routes/+page.svelte

```diff
@@ -26,4 +29,22 @@
 
 <ThemeChange />
 
-<CardBoard {cards} />
+<CardBoard 
+  {cards}
+  on:cardSelected={(data) => {
+    console.log("Card selected:", data.detail)
+    selectedCard = data.detail.card;
+  }}
+  />
+
+<Editor
+  card={selectedCard}
+  on:commit={(data) => {
+    console.log(
+      "Commit card",
+      data.detail.message,
+      data.detail.card
+    );
+    selectedCard = undefined;
+  }}
+  />
```

Comment:
I think it would be best to make this splitscreen, not a drawer, if that makes sense. As it stands, the drawer blocks 50% of the view of the cards, which obscures a lot of helpful info.

I don't think this on the path to the demo, but we'll see.

Reply:
(Ignore gh's comment-line-highlighting here, meant to apply to whole file.)